### PR TITLE
[CHI-394] Add inverse support to Tab component

### PR DESCRIPTION
**Pull request by jllr**
@janibolkvadze 

https://ctl.atlassian.net/browse/CHI-394

### CODE REVIEW

**Comment by allyourbotsarebelongtous** [comment]
Build FAILED. See http://jenkins.elasticbox.com/job/chi_pull_request/365/ for more details.

**Comment by allyourbotsarebelongtous** [comment]
Build FAILED. See http://jenkins.elasticbox.com/job/chi_pull_request/366/ for more details.

File: src/website/views/components/tabs.pug [hunk 1]
```diff
@@ -15,208 +15,445 @@ p.-text
...
+        </li>
+        <li>
+          <a href="#">Tab link</a>
+        </li>
+        <li>
+          <a href="#">Tab link</a>
+        </li>
+      </ul>
 
 h3 Bordered
```
Comment (mattnickles) [inline src/website/views/components/tabs.pug]: @jllr missing support for -inverse. let's set the bottom border color to grey 80 for inverse tabs

**Comment by allyourbotsarebelongtous** [comment]
Build was ABORTED. See http://jenkins.elasticbox.com/job/chi_pull_request/369/ for more details.

**Comment by allyourbotsarebelongtous** [comment]
Build finished SUCCESSFULLY. See http://jenkins.elasticbox.com/job/chi_pull_request/372/ for more details.

**Pull request merged**